versions.props

```diff
@@ -1,4 +1,4 @@
-ch.qos.logback:* = 1.2.11
+ch.qos.logback:* = 1.3.0
```

Comment:
[This PR bumped slf4j-api in conjure](https://github.com/palantir/conjure/pull/1492/files). Versions of logback < 1.3.x are no longer compatible with slf4j-api versions > 2.0.x. [Docs here](https://www.slf4j.org/codes.html#ignoredBindings)

Reply:
The latest 2.x version of Dropwizard (used in our tests) depends on [a version of logback < 1.3.x](https://github.com/dropwizard/dropwizard/blame/v2.1.12/dropwizard-dependencies/pom.xml#L58). This is causing the AutoDeserializeTest cases to fail with:

```
Caused by: java.lang.ClassNotFoundException: ch.qos.logback.classic.jmx.JMXConfigurator
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
```

[The JMX Configurator was removed from logback.](https://logback.qos.ch/news.html#1.3.0:~:text=The%20ch.qos.logback.classic.jmx%20package%20was%20removed%20for%20security%20reasons%20and%20for%20lack%20of%20use.)

I can either:
1. bump our dropwizard dep to the latest version and migrate our tests to use it
1. pin slf4j-api to a version < 2.x
1. see if we can change our tests to not depend on dropwizard.

Going to check how much lift option 1 is

Reply:
Going to pin slf4j-api to pre 2.x version for these tests.